types/serialize-error/serialize-error-tests.ts

```diff
@@ -0,0 +1,5 @@
+import * as serializeError from 'serialize-error';
```

Comment:
An ES6 module namespace object cannot be inviked as a function. Instead use `import serializeError = require('serialize-error');`

Reply:
@armanio123 Fixed